internal/proxy/task_delete.go

```diff
@@ -169,12 +169,12 @@ func (dt *deleteTask) PreExecute(ctx context.Context) error {
 
 	collName := dt.CollectionName
 	if err := validateCollectionName(collName); err != nil {
-		log.Error("Invalid collection name", zap.String("collectionName", collName))
+		log.Error("Invalid collection name", zap.String("collectionName", collName), zap.Error(err))
```

Comment:
critical level：   Info < Warn < Error < Fatal
The high probability of this case is that the user input is illegal, which is not very critical to the system, but critical to the user, since there is an err returned to the client. So it's better to be info or warn here